store/store_test.js

```diff
@@ -13,13 +16,29 @@ let user = {
 };
 
 Feature('store');
-Scenario('test something', ({ I, homePage, authPage, createAccountPage, myAccountPage }) => {
-I.openStore();
-homePage.clickSignIn();
-authPage.fillEmail(Date.now() + '@test.com');
-authPage.clickCreateAccount();
-createAccountPage.fillNewAccountFields(user);
-createAccountPage.clickRegister();
-myAccountPage.verifyPage();
-//I.see('My account');
-}).tag('auth');
+
+Before(({ I }) => {
+    I.openStore();
+});
+
+xScenario('create account', ({ I, homePage, authPage, createAccountPage, myAccountPage }) => {
+    homePage.clickSignIn();
+    authPage.fillRegistrationEmail(Date.now() + '@test.com');
+    authPage.clickCreateAccount();
+    createAccountPage.fillNewAccountFields(user);
+    createAccountPage.clickRegister();
+    myAccountPage.verifyPage();
+}).tag('reg');
+
+Scenario('buy product', async ({ I, homePage, authPage, myAccountPage, productPage, shoppingCartPage }) => {
+    homePage.clickSignIn();
+    authPage.login(user.email, user.password);
```

Comment:
better to use hardcoded login/password parameters, since your `user` object stores data for registration, it should not have login details

Reply:
Done